### ISSUE TITLE
Question about evaluation metrics

### ISSUE THREAD

**Issue by sdulyq** [dc3ea9f/vico_challenge_baseline#22]
I see that the ranking here is based on the score calculated in the script, but this can lead to the ranking results being influenced when one metric is too high. Have you considered optimizing this aspect?

**Comment by sdulyq** [comment]
I have a small question that I would like to ask. The previous approach used the number of optimal metrics to determine the ranking. Why wasn't this approach used this time?

**Comment by dc3ea9f** [comment]
The previous approach introduces biases towards models that perform well in certain metrics but poorly in others, as discussed in point #12. While we aim for a more balanced ranking, where models exhibit relatively balanced performance, it is important that those models which excel in both metrics receive a higher ranking. Therefore, we have decided to explore alternative multi-event scoring methods, such as multiplicative-based (climbing at the 2020 Summer Olympics in Tokyo, Japan) or additive-based ranking methods.

While the multiplicative-based ranking method heavily rewards high finishes and relatively ignores poor finishing results. 
> For instance, if climber A finished 1st, 20th, and 20th and climber B finished 10th, 10th, and 10th, climber B would have a score of 1000 whereas climber A would have a much better score of 400, despite finishing last in 2 out of 3 of the events.
Instead, we choose the additive-based (sum-based) ranking method, which seems better for our task.

Multi-event scoring is a challenging problem, and we are aware that extensive research and discussions have been conducted in the domains of sports and social choices to address this issue. We have dedicated significant efforts to finding a suitable metric that aligns with our competition's requirements. At this stage, we believe that our current metric meets the desired criteria and objectives. However, we remain open to further improvements and adjustments as we continue to evaluate and refine our approach.
